tests/integrationv2/providers.py

```diff
@@ -157,13 +157,16 @@ def setup_client(self):
             cmd_line.append('-r')
 
         cipher_prefs = 'test_all_tls12'
-        if self.options.cipher is Ciphers.KMS_PQ_TLS_1_0_2019_06:
-            cipher_prefs = 'KMS-PQ-TLS-1-0-2019-06'
-        elif self.options.cipher is Ciphers.PQ_SIKE_TEST_TLS_1_0_2019_11:
-            cipher_prefs = 'PQ-SIKE-TEST-TLS-1-0-2019-11'
-        elif self.options.protocol is Protocols.TLS13:
+        if self.options.protocol is Protocols.TLS13:
             cipher_prefs = 'test_all'
 
+        if self.options.cipher is Ciphers.KMS_PQ_TLS_1_0_2019_06 or \
+                self.options.cipher is Ciphers.PQ_SIKE_TEST_TLS_1_0_2019_11 or \
+                self.options.cipher is Ciphers.KMS_PQ_TLS_1_0_2020_07 or \
+                self.options.cipher is Ciphers.KMS_PQ_TLS_1_0_2020_02 or \
+                self.options.cipher is Ciphers.PQ_SIKE_TEST_TLS_1_0_2020_02:
```

Comment:
It might be cleaner to store a list somewhere and just check whether the cipher is in that list.